ios/atb/Info.plist

```diff
@@ -85,6 +87,8 @@
 	<string>Ved å tillate deling av posisjon kan du finne nærmeste holdeplass og planlegge reisen fra din lokasjon</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Ved å tillate deling av posisjon kan du finne nærmeste holdeplass og planlegge reisen fra din lokasjon</string>
+	<key>NSMotionUsageDescription</key>
+	<string>Du gir AtB tillatelse til å bruke bevegelsesdata til analyse og større større forståelse av reisemønster, f.eks. for å se forskjell på sykling, gåing og bilkjøring.</string>
```

Comment:
Will this affect the presentation in App Store for all transport companies?

Reply:
The attribute is there but... if is not requested in the permission prompt, then it never appears, but AtB is the only one that uses it.

Reply:
This is what we have in App Store for the AtB app today. I wondered if adding this would change something there.
<img width="297" alt="Screenshot 2023-10-02 at 13 57 01" src="https://github.com/AtB-AS/mittatb-app/assets/675421/1c45ffa9-c1f1-4d7c-8f14-07d3bd8cc5b2">

Reply:
I don't think so, there is no one related to that motion tracking however is attached to the location as the documentation mention: https://developer.apple.com/documentation/coremotion/cmmotionactivitymanager for Apple and Kettle https://developer.kogenta.com/docs/react-native/usage#ios I do not know how they manage this.